### DISCUSSION TITLE
[2021.10.28] 단어 변환

### DISCUSSION THREAD

**Discussion by hyk339** [pyunji/Hyundai-ITnE-Team_QuadCore#164] [Algorithm]
[단어 변환](https://programmers.co.kr/learn/courses/30/lessons/43163)

**Comment by hyk339** [comment]
[단어 변환 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/programmers/PR_43163.java)

**Comment by pyunji** [comment]
[단어변환 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/PR/PR_43163.java)

**Comment by chaeyeon4u** [comment]
[단어변환 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/PR_43163.java)
